### DISCUSSION TITLE
Full Page Image not working when images are generating

### DISCUSSION THREAD

**Discussion by Kirinin1528** [lllyasviel/stable-diffusion-webui-forge#1728] [Q&A]
I've noticed if you try to click on the preview image to look at the previous batch of images it doesn't work. Clicking doesn't bring up the full page image. It's a little glitchy. Sometimes it does, sometimes no matter how many times you click, it doesn't.

Also after generating an image if you click the button below it to create an upscaled version using your hisres fix settings. Once that image has been generated you can't click on it to view it as a full page image either.

**Comment by aggregate15** [comment]
For me this never worked. Neither in the original A1111, nor in the ForgeUI. I am surprised to know that it works at all for anyone. I thought this was never even an option. Would be good if its available and it can be made to work, though.
